burn-book/src/zero-to-llama2.md

```diff
@@ -0,0 +1 @@
+# Zero to LLaMA2
```

Comment:
I would remove that section for now, but I keep the idea, I will work next week on the book!